core/code/entity_decode.js

```diff
@@ -111,9 +109,23 @@ function detailsPortalData(a) {
   }
 };
 
-function extendedPortalData(a, required) {
+// on getEntities, intel send the history unless the user has never interacted with the portal
+// raw data is either a summary or an extended data with null details
+// when it is a summary, history is forced to 0
+function extendedPortalData(a) {
+  // default value on entities from getEntities
+  if (a.length == SUMMARY_PORTAL_DATA_LENGTH)
+    return {
+      history: parseHistoryDetail(0), // never interacted with this portal
```

Comment:
I suspect that this is wrong. Summary does not contain info and we should return `null`.

Reply:
that's the reverse, when you get portal data there are two cases:
 - from getEntities:
   - as a summary if the portal was never interacted
   - as extended (with null details) if the portal was interacted
 - from getPortalDetails:
   - always as details (no extended data)